src/components/Favorites.js

```diff
@@ -1,21 +0,0 @@
-import React, {Component} from 'react';
```

Comment:
All of these components below were moved to the `Screens` directory and renamed to the appropriate screen. Moving forward, we'll differentiate between a Screen and a Component which can be used within multiple screens.